dockstore-webservice/src/main/java/io/dockstore/webservice/core/Tool.java

```diff
@@ -99,6 +99,7 @@
 
 @Check(constraints = "(toolname NOT LIKE '\\_%')")
 @SuppressWarnings("checkstyle:magicnumber")
+@Schema(name = "DockstoreTool")
```

Comment:
Doesn't this mean the generated client will be `Tool` in Swagger, but `DockstoreTool` in OpenAPI? 

Not a big deal since we're deprecating Swagger, but just wanted to make sure this is intentional.

Reply:
Think it would also be DockstoreTool in Swagger due to `@ApiModel(value = "DockstoreTool",` on liine 68

Reply:
So it is... This is the exact opposite of my concern, it's bringing them into sync. :)